src/app/cli/src/coda_main.ml

```diff
@@ -388,6 +388,30 @@ module type Main_intf = sig
   val receipt_chain_database : t -> Receipt_chain_database.t
 end
 
+module Pending_coinbase = struct
+  open Coda_base.Pending_coinbase
+  include Stable.Latest
```

Comment:
Should not have an `include` here, because `Pending_coinbase.t` should be be `bin_io`.

Reply:
Instead, `type t = Stable.Latest.t [@@deriving ....]`